app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancer.html

```diff
@@ -20,5 +20,6 @@
           ng-click="ctrl.cancel()">Cancel
   </button>
   <submit-button is-disabled="form.$invalid || taskMonitor.submitting"
-                 submitting="taskMonitor.submitting" on-click="ctrl.submit()" is-new="isNew"></submit-button>
+                 submitting="taskMonitor.submitting" on-click="ctrl.submit()"
+                 is-new="isNew" label="submitButtonLabel"></submit-button>
```

Comment:
You can omit the `is-new` attribute here, since it's just used to create a `label` if one isn't passed in.

(It's not obvious - the API on submitButton directive is not very good)